packages/connect-voting/src/helpers/actions.ts

```diff
@@ -0,0 +1,46 @@
+import { utils } from 'ethers'
+import { AppMethod } from "@aragon/connect"
+import { Reward } from '../types'
+
+export const getRewards = (appId: string, fnData: AppMethod): Reward[] => {
```

Comment:
What was your final thought about returning a `Reward` array here? I can't think in which cases we will have multiple rewards on a single action.

Reply:
I was thinking about the possibility of having ac action that calls a contract function that receives a list of recipient addresses and transfer tokens to each of them.

Reply:
That's a good point! Thanks for clarifying it.